packages/application-shell/src/configure-apollo.ts

```diff
@@ -23,30 +18,14 @@ import {
 } from './apollo-links';
 import { getMcApiUrl } from './utils';
 import { isLoggerEnabled } from './utils/logger';
-import version from './version';
-
-declare let window: ApplicationWindow;
 
 type TApolloClientOptions = {
   cache?: InMemoryCacheConfig;
   restLink?: ApolloLink;
 };
 
-const userAgent = createHttpUserAgent({
-  name: 'apollo-client',
-  // version: apolloVersion,
-  libraryName: [window.app.applicationName, 'application-shell'].join('/'),
-  libraryVersion: version,
-  contactUrl: 'https://git.io/fjuyC', // points to the appkit repo issues
-  contactEmail: 'support@commercetools.com',
-});
-
 const httpLink = createHttpLink({
   uri: `${getMcApiUrl()}/graphql`,
-  headers: {
-    accept: 'application/json',
-    'x-user-agent': userAgent,
-  },
```

Comment:
Not needed anymore here, they are now included in the `header-link` and HTTP utils.